infrastructure/ansible/roles/ats_sim/handlers/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+- name: Restart simulator
+  service:
+    name: simulator
+    state: restarted
+    daemon_reload: yes
+  notify: Wait for simulator to become available
+
+- name: Wait for HTTP simulator to become available
+  wait_for:
+    delay: 15
+    sleep: 15
+    timeout: 1200
+    port: "{{ simulator_port }}"
+  listen: Wait for simulator to become available
```

Comment:
It's cleaner to use the task name instead of a listener when only one handler chains to another.